### DISCUSSION TITLE
Deployment of Stirling-PDF on Cloudron

### DISCUSSION THREAD

**Discussion by michael-denzler** [Stirling-Tools/Stirling-PDF#254] [Show and tell]
Hi Anthony,

just to let you know that the Cloudron Community is loving your work on Stirling-PDF. So we did work on a package to be able to deploy your App on Cloudron. (Cloudron is an Application server for selfhosted Applications)

The repository for the package can be found here: https://git.cloudron.io/cloudron/stirling-pdf-app

Best,
Michael

**Comment by Frooodle** [comment]
Let me know if I need to do anything my side to help support it, happy to help

**Comment by Frooodle** [comment]
@michael-denzler  tried to make a fork to update docker file but don't have permissions 

The jar naming changed this release
From spdf to Stirling-pdf.jar 

So docker file needs update

**Reply by michael-denzler** [reply]
Thank you for the proactive information. We will cover this :)
